go.sum

```diff
@@ -13,6 +13,8 @@ github.com/Layr-Labs/cerberus-api v0.0.2-0.20250117193600-e69c5e8b08fd h1:prMzW4
 github.com/Layr-Labs/cerberus-api v0.0.2-0.20250117193600-e69c5e8b08fd/go.mod h1:Lm4fhzy0S3P7GjerzuseGaBFVczsIKmEhIjcT52Hluo=
 github.com/Layr-Labs/eigenda v0.9.0-rc.3.0.20250602235625-e9611587885a h1:VlcjIk2Z1C3E4WQL8IFCg4j1eMiQ71tOGAldiN9Yjac=
```

Comment:
nit: there are a ton of `eigenda` entries here- `go mod tidy` removes all the ones we don't actually need

Reply:
Not sure whether we actually care about this or not

Reply:
[dd2f792](https://github.com/Layr-Labs/eigenda-proxy/pull/411/commits/dd2f792ac7c0230108a8be82b27510c45b855a14)

Reply:
we probably should given it'll bloat the dependency graph unnecessarily

Reply:
tangentially we could add a CI validation once merged into monorepo that runs `go mod tidy` and compares manifests to ensure no diff